### ISSUE TITLE
确保结果的可复现性

### ISSUE THREAD

**Issue by Sakura-not-sleep** [modelscope/ms-swift#3767]
我在微调的时候，每次固定了种子和温度为0.0，具体如下，但是每次跑出来的acc还是不一样，怎么回事呢：

CUDA_VISIBLE_DEVICES=4,5 \
VIDEO_MAX_PIXELS=50176 \
FPS_MAX_FRAMES=12 \
swift sft \
    --model /data1_hdd/j/InternVL2.5/OpenGVLab/InternVL2_5-8B \
    --dataset /data1_hdd/j/InternVL2.5/mydata/train.jsonl \
    --val_dataset /data1_hdd/j/InternVL2.5/mydata/test.jsonl \
    --acc_strategy seq \
    --train_type lora \
    --torch_dtype bfloat16 \
    --attn_impl flash_attn \
    --num_train_epochs 5 \
    --per_device_train_batch_size 2 \
    --split_dataset_ratio 0.0 \
    --per_device_eval_batch_size 1 \
    --learning_rate 8e-5 \
    --lora_rank 8 \
    --lora_alpha 32 \
    --target_modules all-linear \
    --freeze_vit true \
    --freeze_aligner true \
    --gradient_accumulation_steps 4 \
    --save_strategy epoch \
    --eval_strategy epoch \
    --eval_steps 1 \
    --save_steps 1 \
    --save_total_limit 5 \
    --logging_steps 10 \
    --max_length 2048 \
    --modules_to_save classifier \
    --output_dir /data1_hdd/j/InternVL2.5/add_8B \
    --warmup_ratio 0.1 \
    --seed 2025 \
    --temperature 0 \
    --dataloader_num_workers 4
